src/components/real/communication/wings_command_sender_to_c2a.cpp

```diff
@@ -0,0 +1,147 @@
+/*
+ * @file wings_command_sender_to_c2a.cpp
+ * @brief A component to send C2A command using WINGS operation file
+ * @note Used C2A functions: CCP_register_rtc, CCP_register_tlc
+ *       Used C2A structures: CMD_CODE
+ */
+
+#include "wings_command_sender_to_c2a.hpp"
+
+#include <library/initialize/initialize_file_access.hpp>
+#include <library/utilities/macros.hpp>
+#include <regex>
+
+#ifdef USE_C2A
+#if C2A_CORE_VER_MAJOR == 4
+// c2a-core v4
+#include "src_core/TlmCmd/common_cmd_packet_util.h"
```

Comment:
c2a-core v4 ではディレクトリ名が変わっています
```suggestion
#include "src_core/tlm_cmd/common_cmd_packet_util.h"
```

Reply:
https://github.com/arkedge/c2a-core/tree/v4.0.0-beta.0

この修正を行なった時には、こちらしかなかったので、TlmCmdでしたが、レビューを待っている間（先週）に新しくリリースbeta.1がされたようですね。

Reply:
よくみたら、beta.0はyankされてしまっているようですね。

Reply:
そうです．そして `arkedge/c2a-core` v4.0.0-beta.0 の yank などもあってゴタゴタしていたのもあってお知らせが遅くなってしまっていました......